### PR TITLE
Add release notes for Kyma 1.10 Quebec

### PR DESCRIPTION
**Pull request by kazydek**
**Description**

Changes proposed in this pull request:

- Created release notes for the upcoming Kyma 1.10 Quebec release.

**Related issue(s)**
See also #536

### CODE REVIEW

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+### Uninstall command removed
+
+The `uninstall` command was one of the first commands available in Kyma CLI. It was based on a very simple implementation that turned out to work unreliably. Since we want to redesign it but there are other priorities at the moment, we decided to remove the command for now and restore it only when this feature is stabilized.
+
+## Compass
+
+### Runtime Provisioner registers Runtimes in the Director
+
+With the 1.10 release, the default flow no longer requires manual registration of Runtimes in the Director – it is now done automatically by the Runtime Provisioner when the request for Runtime provisioning is received. Minor API changes are present here – the mutation now requires input parameters for the Runtime registration.
```
Comment (bszwarc) [inline content/blog-posts/2020-02-06-release-notes-1.10/index.md]: Do we know what received the Runtime request? Then you can use active voice here to say `when [sth] receives the request for Runtime provisioning`.

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+### Custom configuration of Runtimes
+
+The new functionality was introduced in the Runtime Provisioner's API. Starting from this release, you can specify which components you would like to install on your Runtime. The Runtime Provisioner's API now accepts a list of components (with override values) that are passed onto the Kyma Installer. This is used to create the Installation CR and should contain only components that are already available in Kyma.
```
Comment (bszwarc) [inline content/blog-posts/2020-02-06-release-notes-1.10/index.md]:
```suggestion
The new functionality was introduced in the Runtime Provisioner's API. Starting from this release, you can specify which components you would like to install on your Runtime. The Runtime Provisioner's API now accepts a list of components (with override values) that are passed on to the Kyma Installer. This is used to create the Installation CR and should contain only components that are already available in Kyma.
```
or just `to` ?

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+- `OAuth2` for app-to-app access management
+- `JWT` for authenticated users only
+
+![API rules 2](./api-rules2.png)
+
+## Eventing
+
+### Knative Kafka Channel integration
+
+Thanks to Knative Eventing that allows you to provide implementations of various channels for event forwarding, Kyma now comes with an optional [Knative Kafka channel](https://github.com/kyma-incubator/knative-kafka) that was tested against Azure Event Hubs. You can read how to use it in the Knative Eventing Mesh [here](https://kyma-project.io/docs/components/event-bus/#details-knative-eventing-mesh-alpha-kafka).
```
Comment (bszwarc) [inline content/blog-posts/2020-02-06-release-notes-1.10/index.md]:
```suggestion
Thanks to Knative Eventing allowing you to provide implementations of various channels for event forwarding, Kyma now comes with an optional [Knative Kafka channel](https://github.com/kyma-incubator/knative-kafka) that was tested against Azure Event Hubs. You can read how to use it in the Knative Eventing Mesh [here](https://kyma-project.io/docs/components/event-bus/#details-knative-eventing-mesh-alpha-kafka).
```

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+  name: "Karolina Zydek, Technical Writer @Kyma"
+tags:
+  - release-notes
+type: release
+releaseTag: "1.10.0"
+redirectFrom:
+  - "/blog/release-notes-110"
+---
+
+The journey all the way from Paris 1.9 to Quebec 1.10 turned out to be quite extensive but rewarding for the Kyma crew. We reached Quebec with renewed batteries, leveling them up to Kubernetes 1.16, Istio 1.4.3, Minikube 1.6, and Velero 1.2. We created production profiles for Istio and Ory, made a few tweaks in Compass, and introduced native DNS support for Kyma provisioned on Gardener clusters. Finally, we managed to add such optional features as the API Rules view in the Console UI or the new Knative Kafka channel for event forwarding. For this news and many more, read the full story behind Kyma 1.10 Quebec.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply content/blog-posts/2020-02-06-release-notes-1.10/index.md]: "them" meaning batteries - I mean "renewal" as upgrades, not sure if recharging matches the context but, also, not sure if that matters altogether so much so if my original intention here is not that clear I may as well change it.

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+![Application template](./application-template.png)
+
+## Console
+
+###  Creation of Access Rules for your APIs from the Console UI
+
+Kyma comes with a built-in API Gateway Controller that allows you to define API Rules for your services. With this release, it is now possible to define those API Rules from the Console UI.
+
+This view is still considered experimental. You can use it only after enabling **Experimental functionalities** under **General Settings**.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply content/blog-posts/2020-02-06-release-notes-1.10/index.md]: ok

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+
+All of those steps are now done in an automated fashion, without any manual configuration required on your end.
+
+### Installation of components from remote locations
+
+The Kyma Installer has a new feature allowing you to install your components from remote locations that are either GitHub repositories or HTTP servers. Previously, in order to install a new component using Installer, you had to build the Installer on your own. Currently, the Installation CR contains the `source.url` field that allows you to specify the remote location of your Helm chart. The Installer tries to fetch sources from this location three times before the installation or upgrade fails. To learn more, see our [docs](https://kyma-project.io/docs/root/kyma/#configuration-install-components-from-user-defined-ur-ls).
+
+### Kubernetes 1.16 compatibility
+
+Kubernetes 1.16 provides new functionalities but also drops support for old API versions. During this release, we migrated the Kubernetes API versions used by Kyma to the latest ones to meet the compatibility requirements. Thanks to this, it is now possible to install Kyma on Kubernetes 1.16.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply content/blog-posts/2020-02-06-release-notes-1.10/index.md]: or rather `features`? Would that be better?

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+releaseTag: "1.10.0"
+redirectFrom:
+  - "/blog/release-notes-110"
+---
+
+The journey all the way from Paris 1.9 to Quebec 1.10 turned out to be quite extensive but rewarding for the Kyma crew. We reached Quebec with renewed batteries, leveling them up to Kubernetes 1.16, Istio 1.4.3, Minikube 1.6, and Velero 1.2. We created production profiles for Istio and Ory, made a few tweaks in Compass, and introduced native DNS support for Kyma provisioned on Gardener clusters. Finally, we managed to add such optional features as the API Rules view in the Console UI or the new Knative Kafka channel for event forwarding. For this news and many more, read the full story behind Kyma 1.10 Quebec.
+
+<!-- overview -->
+
+>**CAUTION:** Before upgrading your Kyma deployment to 1.10, you must upgrade Tiller. Read the [**Migrations and upgrades**](#migrations-and-upgrades) section carefully for more details. If you upgrade to a newer release without performing the steps described in that section, you can compromise the functionality of your cluster or make it unusable altogether.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply content/blog-posts/2020-02-06-release-notes-1.10/index.md]: skipped it 👍

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+
+## Service Management
+
+### ServiceInstance creation successful but not visible
+
+If your ServiceInstance creation was successful and yet the release is marked as `FAILED` on the releases list when running the `helm list` command, it means that there is an error on the Helm's side that was not passed onto the Helm Broker. To get the error details, check Tiller's logs.
+
+### Helm Broker blocks the deletion of (Cluster)ServiceBrokers with dependencies
+
+If a ServiceBroker or ClusterServiceBroker custom resource is deleted before the removal of the corresponding ServiceInstance, the ServiceInstance cannot be deprovisioned from the Service Catalog. As of this release, Helm Broker doesn't remove the (Cluster)ServiceBroker custom resource if there are any ServiceInstances created using this broker. The Helm Broker will remove the (Cluster)ServiceBroker custom resource only when there are no related ServiceInstances remaining.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply content/blog-posts/2020-02-06-release-notes-1.10/index.md]: I will change it but only partially, as consulted with @klaudiagrz

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+
+### Updated Velero and improved Azure support
+
+In order to overcome issues related to AKS-based environments, we updated the Velero version to 1.2. The purpose of the upgrade was to ensure that the backup and restore procedure works smoothly on AKS clusters.
+
+## CLI
+
+### Support for Minikube 1.6
+
+CLI and Kyma got tested with the latest Minikube version 1.6 that has now become the officially supported version. Try it out on your own.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply content/blog-posts/2020-02-06-release-notes-1.10/index.md]: I'll remove it.

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md (hunk 1 above)
Comment (kazydek) [reply content/blog-posts/2020-02-06-release-notes-1.10/index.md]: Will ckeck that.

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md (hunk 3 above)
Comment (kazydek) [reply content/blog-posts/2020-02-06-release-notes-1.10/index.md]: I would leave it as it is if you don't mind.

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md (hunk 1 above)
Comment (kazydek) [reply content/blog-posts/2020-02-06-release-notes-1.10/index.md]: Checked and that is done by the Runtime Provisioner.

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md [hunk 10]
```diff
@@ -0,0 +1,155 @@
...
+
+## Service Management
+
+### ServiceInstance created but not visible
+
+If your ServiceInstance creation was successful and yet the release is marked as `FAILED` on the releases list when running the `helm list` command, it means that there is an error on the Helm's side that was not passed to the Helm Broker. To get the error details, check Tiller's logs.
+
+### Helm Broker blocks deleting (Cluster)ServiceBrokers with dependencies
+
+If a ServiceBroker or ClusterServiceBroker custom resource (CR) is deleted before you remove the corresponding ServiceInstance, the ServiceInstance cannot be deprovisioned from the Service Catalog. As of this release, Helm Broker doesn't remove the (Cluster)ServiceBroker CR if there are any ServiceInstances in the cluster provisioned by the Helm Broker. The Helm Broker will remove the (Cluster)ServiceBroker CR only when there are no related ServiceInstances remaining.
```
Comment (majakurcius) [inline content/blog-posts/2020-02-06-release-notes-1.10/index.md]:
```suggestion
If a ServiceBroker or ClusterServiceBroker custom resource (CR) is deleted before you remove the corresponding ServiceInstance, the ServiceInstance cannot be deprovisioned from the Service Catalog. As of this release, Helm Broker does not remove the (Cluster)ServiceBroker CR if there are any ServiceInstances in the cluster provisioned by the Helm Broker. The Helm Broker will remove the (Cluster)ServiceBroker CR only when there are no related ServiceInstances remaining.
```
Please ignore it if you like yours better.

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md [hunk 11]
```diff
@@ -0,0 +1,155 @@
...
+
+The latest release comes with a new version of Istio 1.4.3. This change has affected our Istio installation as now it is installed using [istioctl](https://istio.io/docs/reference/commands/istioctl/). The reason behind upgrading Kyma to the latest Istio version was that the support for 1.3 will end [soon](https://istio.io/news/support/announcing-1.3-eol/). To read more about all changes introduced in 1.4.3, see [Istio release notes](https://istio.io/news/releases/1.4.x/announcing-1.4.3/).
+
+### Production profile for Istio
+
+The default cluster configuration provides Istio that is good enough for playground purposes but not sufficient for a production-grade workload. This release comes with the production profile for Istio which increases application scalability (updated HPA) and provides higher CPU and memory limits/requests for the Istio control plane components. These changes improve the overall Istio components' performance and stability. The production profile configuration is optional, not enabled by default. To learn more about the profile, read our [docs](https://kyma-project.io/docs/components/service-mesh/#configuration-service-mesh-production-profile).
```
Comment (majakurcius) [inline content/blog-posts/2020-02-06-release-notes-1.10/index.md]:
```suggestion
The default cluster configuration provides Istio that is good enough for playground purposes but not sufficient for a production-grade workload. This release comes with the production profile for Istio which increases Application scalability (updated HPA) and provides higher CPU and memory limits/requests for the Istio control plane components. These changes improve the overall Istio components' performance and stability. The production profile configuration is optional, not enabled by default. To learn more about the profile, read our [docs](https://kyma-project.io/docs/components/service-mesh/#configuration-service-mesh-production-profile).
```
?

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md [hunk 12]
```diff
@@ -0,0 +1,155 @@
...
+
+### Production profile for Ory
+
+The latest release also ships with the production profile for Ory. This involves increased CPU requests for the Oathkeeper and an automatically deployed PostgreSQL database in which Hydra stores data. This profile isn't enabled by default. Currently, if you enable the production profile during the Kyma upgrade, you will need to manually synchronize clients in Hydra. We will be working on this synchronization mode in the upcoming release.
```
Comment (majakurcius) [inline content/blog-posts/2020-02-06-release-notes-1.10/index.md]:
```suggestion
The latest release also ships with the production profile for Ory. This involves increased CPU requests for the Oathkeeper and an automatically deployed PostgreSQL database in which Hydra stores data. This profile isn't enabled by default. Currently, if you enable the production profile during the Kyma upgrade, you will need to manually synchronize clients in Hydra. We will be working on this synchronization mode during the next release cycle.  
```

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md [hunk 13]
```diff
@@ -0,0 +1,155 @@
...
+
+### AKS metrics
+
+Current Monitoring deployment does not allow Prometheus to scrape metrics, such as CPU, memory, and Network, from kubelet on AKS. As a consequence, metrics are not displayed in Grafana. To solve this problem, you need to apply the patch for the `monitoring-kubelet` ServiceMonitor. For detailed instructions, see [this](https://github.com/kyma-project/kyma/issues/6846) issue.
+
+## Backup
+
+### Updated Velero and improved Azure support
+
+In order to overcome issues related to AKS-based environments, we updated the Velero version to 1.2. The purpose of the upgrade was to ensure that the backup and restore procedure works smoothly on AKS clusters.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply content/blog-posts/2020-02-06-release-notes-1.10/index.md]: I'll take the first part of the suggestion :)

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md [hunk 14]
```diff
@@ -0,0 +1,155 @@
...
+
+## CLI
+
+### Support for Minikube 1.6
+
+CLI and Kyma got tested with the latest Minikube version 1.6 that has now become the officially supported version.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply content/blog-posts/2020-02-06-release-notes-1.10/index.md]: were?

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md (hunk 10 above)
Comment (kazydek) [reply content/blog-posts/2020-02-06-release-notes-1.10/index.md]: I tried to contract whatever possible in these release notes so I will stick to it in here to be consistent throughout the doc.

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md [hunk 15]
```diff
@@ -0,0 +1,155 @@
...
+  name: "Karolina Zydek, Technical Writer @Kyma"
+tags:
+  - release-notes
+type: release
+releaseTag: "1.10.0"
+redirectFrom:
+  - "/blog/release-notes-110"
+---
+
+The journey all the way from Paris 1.9 to Quebec 1.10 turned out to be quite extensive but rewarding for the Kyma crew. We reached Quebec with recharged batteries, leveling them up to Kubernetes 1.16, Istio 1.4.3, Minikube 1.6, and Velero 1.2. We created production profiles for Istio and Ory, made a few tweaks in Compass, and introduced native DNS support for Kyma provisioned on Gardener clusters. Finally, we managed to add a few optional features, such as the API Rules view in the Console UI or the new Knative Kafka channel for event forwarding. For this news and many more, read the full story behind Kyma 1.10 Quebec.
```
Comment (klaudiagrz) [inline content/blog-posts/2020-02-06-release-notes-1.10/index.md]:
```suggestion
The journey all the way from Paris 1.9 to Quebec 1.10 turned out to be quite extensive but rewarding for the Kyma crew. We reached Quebec with recharged batteries, leveling them up to Kubernetes 1.16, Istio 1.4.3, Minikube 1.6, and Velero 1.2. We created production profiles for Istio and ORY, made a few tweaks in Compass, and introduced native DNS support for Kyma provisioned on Gardener clusters. Finally, we managed to add a few optional features, such as the API Rules view in the Console UI or the new Knative Kafka channel for event forwarding. For this news and many more, read the full story behind Kyma 1.10 Quebec.
```

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md [hunk 16]
```diff
@@ -0,0 +1,155 @@
...
+
+<!-- overview -->
+
+>**CAUTION:** Before upgrading your Kyma deployment to 1.10, you must upgrade Tiller. Read the [**Migrations and upgrades**](#migrations-and-upgrades) section carefully for more details. If you upgrade to a newer release without performing the steps described in that section, you may compromise the functionality of your cluster or make it unusable.
+
+See the overview of all changes in this release:
+
+- [Migrations and upgrades](#migrations-and-upgrades) - Tiller upgrade required before upgrading Kyma to 1.10
```
Comment (klaudiagrz) [inline content/blog-posts/2020-02-06-release-notes-1.10/index.md]: Good to have it on top of RNs! GJ! 👏

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md [hunk 17]
```diff
@@ -0,0 +1,155 @@
...
+- [Known issues](#known-issues) - Monitoring-related issues with AKS metrics
+- [Backup](#backup) - Updated Velero and improved Azure support
+- [CLI](#cli) - Support for Minikube 1.6, removal of the `uninstall` command
+- [Compass](#compass) - Automatic Runtime registration in the Director, custom configuration of Runtimes, automatic Runtime-Director connection, new templates for Application registration
+- [Console](#console) - Possibility to create Access Rules for APIs from the Console UI
+- [Eventing](#eventing) - Knative Kafka Channel integration, new Grafana dashboards for the Knative Eventing Mesh
+- [Installation](#installation) - Native Gardener DNS support, installation of components from remote locations, Kubernetes 1.16 compatibility
+- [Logging](#logging) - Support for Fluent Bit as the new log collector
+- [Service Management](#service-management) - ServiceInstance created but not visible, Helm Broker blocks deleting (Cluster)ServiceBrokers with dependencies
+- [Service Mesh](#service-mesh) - Istio upgraded to 1.4.3, production profile for Istio, production profile for Ory
```
Comment (klaudiagrz) [inline content/blog-posts/2020-02-06-release-notes-1.10/index.md]:
```suggestion
- [Service Mesh](#service-mesh) - Istio upgraded to 1.4.3, production profile for Istio, production profile for ORY
```

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md [hunk 18]
```diff
@@ -0,0 +1,155 @@
...
+- [Migrations and upgrades](#migrations-and-upgrades) - Tiller upgrade required before upgrading Kyma to 1.10
+- [Known issues](#known-issues) - Monitoring-related issues with AKS metrics
+- [Backup](#backup) - Updated Velero and improved Azure support
+- [CLI](#cli) - Support for Minikube 1.6, removal of the `uninstall` command
+- [Compass](#compass) - Automatic Runtime registration in the Director, custom configuration of Runtimes, automatic Runtime-Director connection, new templates for Application registration
+- [Console](#console) - Possibility to create Access Rules for APIs from the Console UI
+- [Eventing](#eventing) - Knative Kafka Channel integration, new Grafana dashboards for the Knative Eventing Mesh
+- [Installation](#installation) - Native Gardener DNS support, installation of components from remote locations, Kubernetes 1.16 compatibility
+- [Logging](#logging) - Support for Fluent Bit as the new log collector
+- [Service Management](#service-management) - ServiceInstance created but not visible, Helm Broker blocks deleting (Cluster)ServiceBrokers with dependencies
```
Comment (klaudiagrz) [inline content/blog-posts/2020-02-06-release-notes-1.10/index.md]:
```suggestion
- [Service Management](#service-management) - the possible issue of created ServiceInstance with the failed status, Helm Broker blocks deleting (Cluster)ServiceBrokers with dependencies
```

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md [hunk 19]
```diff
@@ -0,0 +1,155 @@
...
+
+## Logging
+
+### Support for Fluent Bit as the new log collector
+
+To enable more integrations with external logging systems, the Promtail log collector that came together with Loki got replaced with [Fluent Bit](https://fluentbit.io/). Fluent Bit is a well-known component in the Kubernetes ecosystem and provides many integrations with other systems out of the box. The integration with the bundled Loki is still assured as part of Kyma's Logging component.
+
+## Service Management
+
+### ServiceInstance created but not visible
```
Comment (klaudiagrz) [inline content/blog-posts/2020-02-06-release-notes-1.10/index.md]:
```suggestion
### The possible issue of created ServiceInstance with the failed status
```

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md [hunk 20]
```diff
@@ -0,0 +1,155 @@
...
+
+### Production profile for Istio
+
+The default cluster configuration provides Istio that is good enough for playground purposes but not sufficient for a production-grade workload. This release comes with the production profile for Istio which increases Application scalability (updated HPA) and provides higher CPU and memory limits/requests for the Istio control plane components. These changes improve the overall Istio components' performance and stability. The production profile configuration is optional, not enabled by default. To learn more about the profile, read our [docs](https://kyma-project.io/docs/components/service-mesh/#configuration-service-mesh-production-profile).
+
+### Production profile for Ory
```
Comment (klaudiagrz) [inline content/blog-posts/2020-02-06-release-notes-1.10/index.md]:
```suggestion
### Production profile for ORY
```

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md [hunk 21]
```diff
@@ -0,0 +1,155 @@
...
+
+The latest release also ships with the production profile for Ory. This involves increased CPU requests for the Oathkeeper and an automatically deployed PostgreSQL database in which Hydra stores data. This profile isn't enabled by default. Currently, if you enable the production profile during the Kyma upgrade, you will need to manually synchronize clients in Hydra. We will be working on this synchronization mode during the next release cycle.
```
Comment (klaudiagrz) [inline content/blog-posts/2020-02-06-release-notes-1.10/index.md]:
```suggestion
The latest release also ships with the production profile for ORY. This involves increased CPU requests for the Oathkeeper and an automatically deployed PostgreSQL database in which Hydra stores data. This profile isn't enabled by default. Currently, if you enable the production profile during the Kyma upgrade, you will need to manually synchronize clients in Hydra. We will be working on this synchronization mode during the next release cycle.  
```

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md (hunk 18 above)
Comment (kazydek) [reply content/blog-posts/2020-02-06-release-notes-1.10/index.md]: How about: `Possible failed status for created ServiceInstances`?

File: content/blog-posts/2020-02-06-release-notes-1.10/index.md (hunk 19 above)
Comment (kazydek) [reply content/blog-posts/2020-02-06-release-notes-1.10/index.md]: as above?

**Pull request merged**